### ISSUE TITLE
Question - How to overwrite .auto.txt files?

### ISSUE THREAD

**Issue by wasciutto** [nipy/heudiconv#611]
This question could be expanded to a general question of "what is the right way to iterate over a heuristic.py file?", for which there may be a bigger picture that I am missing.

What I mean by iteration over a heuristic file - Let's say I run heudiconv on a specific subject, getting a .heudiconv folder in my BIDS output. As is the case 95% of the time, let's also say I don't get my conversion right on the first try, and some of the mappings in `.heudiconv/*/info/*.auto.txt` are incorrect.

The issue I'm having is that if I rewrite my heudiconv.py file to change how these mappings are made, and then rerun heudiconv, these `.auto.txt` files will not update. The `--overwrite` flag will not change this fact, as it only seems to apply to the actual images in the main BIDS directory. 

So what I end up having to do, and have seen others do, is either deleting the `.heudiconv` directory any subjects I want to rerun, or just creating an entirely new BIDS directory to make sure that the "caching" heudiconv does with those mappings is not hiding something that should be working.

Is there a better way to iterate over a heudiconv file that I am missing?

Thanks!

-Will

### Platform details:

Choose one:
- [ ] Local environment

- [ ] Container

- Heudiconv version:

**Comment by yarikoptic** [comment]
yeah, IIRC we do not have a flag to "override" those files. I usually just manually `rm -r` that `.heudiconv` directory.  I guess it might make sense to have more "specific" `--override` option which would become not a flag but take values like `data`, `heudiconv-auto`, `all`.

**Comment by yarikoptic** [comment]
It also relates to https://github.com/nipy/heudiconv/issues/84 (and references to it there in) where there is a desire for magical "override" of those files if there were some updates to heuristic/env/files... not sure if we should pursue the magic or while breaking CLI (#620 ) also change how `heudiconv` operates in "two stage" mode and make two stages explicit, thus overriding prior mapping etc if stage one is explicitly requested.
